RadialGM.pro.shared

```diff
@@ -44,4 +44,4 @@
             <value type="bool" key="EditorConfiguration.inEntireDocument">true</value>
         </valuemap>
     </data>
-</qtcreator>
+</qtcreator>
```

Comment:
You forget the end of file newline. Please add it, this is a standard convention originating from Unix.